src/reducers/flavors.js

```diff
@@ -0,0 +1,48 @@
+import { buildActions } from 'utils';
+
+export const types = buildActions('flavors', [
```

Comment:
My recipe editor PR also adds one of these, but I'll rebase mine off yours and deal with the conflicts.